printer-confs/base.cfg

```diff
@@ -59,19 +59,18 @@ gcode:
     Part_Light_ON
     G92 E0 
     G90                       ; Use absolute coordinates
-    BED_MESH_CLEAR
-    G28     
```

Comment:
It may be better to home, then move head up by 50mm, and then start to heat the bed. Otherwise we don't know where the head is, and it may be heating up from the bed if it is too low